depends/packages/tl_expected.mk

```
@@ -1,6 +1,6 @@
 package=tl_expected
 $(package)_version=1.0.1
-$(package)_download_path=https://github.com/TartanLlama/expected/archive/refs/tags
+$(package)_download_path=https://github.com/TartanLlama/expected/archive
 $(package)_download_file=96d547c03d2feab8db64c53c3744a9b4a7c8f2c5.tar.gz
 $(package)_file_name=tl-expected-1.0.1.tar.gz
```

```suggestion
$(package)_download_file=$($(package)_version).tar.gz
$(package)_file_name=$(package)_$($(package)_version).tar.gz
```
No 1.0.1 version has been released, and it would be at a different commit if it were.